docs/data/toolpad/getting-started/quickstart.md

```diff
@@ -0,0 +1,147 @@
+# Quickstart
+
+<p class="description">Learn the fundamentals of building with Toolpad by creating and deploying a simple app.</p>
```

Comment:
In terms of nomenclature I see "app" being used as well as "application". Shall we standardize on "application", (or "Toolpad application" when we need to be more specific)

```suggestion
<p class="description">Learn the fundamentals of building with Toolpad by creating and deploying a simple application.</p>
```